src/Services/QuestionService.php

```diff
@@ -30,8 +32,8 @@ public function deleteQuestion(Question $question): bool
             });
 
             return true;
-        } catch (\Exception $err) {
-            return false;
+        } catch (Exception $err) {
+            throw new UnprocessableEntityHttpException(__('Question deleted failed'));
```

Comment:
Why catch here if all you are doing is throwing again? And throwing Http exception from Service and not Request/Controller is probably bad, what if this service was used in a Job/Queue/EventListener (something that happens outside of HTTP context)

Reply:
done